common/engine/keyboardprocessor/tests/unit/kmx/041 - long context and deadkeys.kmn

```diff
@@ -0,0 +1,23 @@
+c Description: Tests that we don't split a dk in context:set
+c keys: [K_1][K_2][K_3]
+c context: \u00ffff\u000008\u00000001abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh
+c expected: \u00ffff\u000008\u00000001abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh123
+
+store(&version) '6.0'
+
+begin Unicode > use(Main)
+
+group(Main) using keys
+
+c Test 1 
+dk(1) 'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh' + '1' > context '1'
+
+c Test 2
```

Comment:
```suggestion
c Test 2: test that we haven't split a deadkey code: `UC_SENTINEL CODE_DEADKEY <deadkey_number>`
c   because: `CODE_DEADKEY` = `U+0008`; `deadkey_number` = `U+0001`
```